### PR TITLE
Prepared package for 3.3 release

### PR DESCRIPTION
**Pull request by alongosz**
This PR prepares the package for 3.3 release.

Changes
- [x] Added LICENSE and LICENSE-bul files
- [x] Added simplified README
- [x] Added COPYRIGHT file
- [x] Fixed `composer.json` description to properly refer to Ibexa Open Source
- [x] Forced installation of Ibexa packages from stable dev branches despite [`prefer-stable` in the website skeleton](https://github.com/ibexa/website-skeleton/blob/cf298380ad68f4903b39215af319088af1c33f16/composer.json#L7)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+1. Create a new project from [ibexa/website-skeleton](https://github.com/ibexa/website-skeleton).
+    ```
+    composer create-project ibexa/website-skeleton ibexa_website
+    ```
+1. Require `ibexa-oss` package
+    ```
+    composer require ibexa-oss
+    ```
+
+Learn more about our commercial Product - [Ibexa DXP](https://www.ibexa.co/products).
```
Comment (DominikaK) [inline README.md:16]:
```suggestion
Learn more about our commercial product - [Ibexa DXP](https://www.ibexa.co/products).
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# Ibexa Open Source Symfony Flex metapackage
+
+This is a metapackage for Symfony Flex-based Ibexa Open Source installation.
+
+## Installation
+
+1. Create a new project from [ibexa/website-skeleton](https://github.com/ibexa/website-skeleton).
```
Comment (DominikaK) [inline README.md:7]:
```suggestion
1. Create a new project from [ibexa/website-skeleton](https://github.com/ibexa/website-skeleton):
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    ```
+    composer create-project ibexa/website-skeleton ibexa_website
+    ```
+1. Require `ibexa-oss` package
```
Comment (DominikaK) [inline README.md:11]:
```suggestion
1. Require `ibexa-oss` package:
```

**Review by lserwatka — APPROVED** [review]

**Review by DominikaK — APPROVED** [review]

File: composer.json [hunk 4]
```diff
@@ -2,31 +2,31 @@
     "name": "ibexa/oss",
     "type": "metapackage",
     "license": "GPL-2.0-only",
```
Comment (andrerom) [inline composer.json:4]:
```suggestion
    "license": "(GPL-2.0-only or proprietary)",
```

File: .github/pull_request_template.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+| **Target eZ Platform version** | `v3.x` - please update `x` accordingly
+| **BC breaks**                          | yes/no
+| **Doc needed**                       | yes/no
+
+<!-- Replace this comment with Pull Request description -->
+
+#### Checklist:
+- [ ] Provided PR description.
+- [ ] Tested the solution manually.
+- [ ] Provided automated test coverage.
```
Comment (webhdx) [inline .github/pull_request_template.md:14]: Doesn't apply to this repository.

File: composer.json (hunk 4 above)
Comment (alongosz) [reply composer.json]: Thx. Updated and added LICENSE-bul via ca55b22.

File: .github/pull_request_template.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+- [ ] Checked that target branch is set correctly (master for features, the oldest supported for bugs).
```
Comment (webhdx) [inline .github/pull_request_template.md:15]: This should be probably rephrased as this repo won't really handle any bugfixes.

**Review by webhdx — APPROVED** [review]
I have only small complaint about PR template. Apart from that everything looks good.

File: .github/pull_request_template.md (hunk 5 above)
Comment (alongosz) [reply .github/pull_request_template.md]: Dropped the line

File: .github/pull_request_template.md (hunk 6 above)
Comment (alongosz) [reply .github/pull_request_template.md]: Dropped parenthesis

**Comment by alongosz** [comment]
Small update: added also COPYRIGHT file based on what Satis does. This should now fully reflect legal requirements I hope

**Review by webhdx — APPROVED** [review]

**Review by andrerom — APPROVED** [review]

**Review by lserwatka — APPROVED** [review]

**Comment by alongosz** [comment]
Update: baf5bb4 Bumped packages versions to the latest stable installed - to avoid installed older versions and for composer mem. optimization.

**Pull request merged**